test/higher_order_ops/test_invoke_subgraph.py

```diff
@@ -292,19 +309,25 @@ def forward(self, l_x_: "f32[8]", l_y_: "f32[8]"):
             """\
 class GraphModule(torch.nn.Module):
     def forward(self, primals_1: "f32[8]", primals_2: "f32[8]"):
-        repeated_subgraph0 = self.repeated_subgraph0
-        invoke_subgraph = torch.ops.higher_order.invoke_subgraph(repeated_subgraph0, '___forward_invoke_subgraph_0', (primals_1, primals_2));  repeated_subgraph0 = None
-        getitem: "f32[8]" = invoke_subgraph[0];  invoke_subgraph = None
-
-        repeated_subgraph0_1 = self.repeated_subgraph0
-        invoke_subgraph_1 = torch.ops.higher_order.invoke_subgraph(repeated_subgraph0_1, '___forward_invoke_subgraph_0', (getitem, primals_2));  repeated_subgraph0_1 = None
-        getitem_1: "f32[8]" = invoke_subgraph_1[0];  invoke_subgraph_1 = None
-        return (getitem_1, primals_1, primals_2, getitem)
-
-    class repeated_subgraph0(torch.nn.Module):
-        def forward(self, arg0_1: "f32[8]", arg1_1: "f32[8]"):
-            mul: "f32[8]" = torch.ops.aten.mul.Tensor(arg0_1, arg1_1);  arg0_1 = arg1_1 = None
-            return (mul,)
+        ___forward_invoke_subgraph_0_post_graph = self.___forward_invoke_subgraph_0_post_graph
```

Comment:
Do you have a test that shows a backward graph and how the backward graph got partitioned?

Also mul is probably not the best example because the partition would recompute it (due to it being a pointwise operation), so we wouldn't be able to tell if this PR is working. torch.mm would be better

Reply:
Yea, added `test_bwd_partitioning`